doc/nrf/libraries/others/contin_array.rst

```diff
@@ -0,0 +1,25 @@
+.. _lib_contin_array:
+
+Continuous array
+################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. TODO: Add description
+
+Configuration
+*************
+
+To enable the library, set the :kconfig:option:`CONFIG_CONTIN_ARRAY` Kconfig option to ``y`` in the project configuration file :file:`prj.conf`.
```

Comment:
For now, there is no such setting. The functionality is simply available, but I may be corrected on that in the original PR.

Reply:
So how can you enable the library, if it is missing its Kconfig?

Reply:
It is always available.